src/v2/stack_aggregator.py

```diff
@@ -361,3 +369,181 @@ def execute(request: Dict, persist=True):
         return {'aggregation': 'success',
                 'external_request_id': output.external_request_id,
                 'result': output_dict}
+
+
+class GoAggregator(Aggregator):
+    """GoAggregator is a Superset of Aggregator with Golang Added func."""
+
+    def __init__(self, request: StackAggregatorRequest = None,
+                 normalized_packages: GoNormalizedPackages = None):
+        """Initialize common fields."""
+        super().__init__(request, normalized_packages)
+        self._normalized_packages = normalized_packages
+        self.filtered_vul = {}
+
+    def _get_package_details_with_vulnerabilities(self) -> List[Dict[str, object]]:
+        """Get package data from graph along with vulnerability."""
+        time_start = time.time()
+        pkgs_with_vuln = {
+            "result": {
+                "data": []
+            }
+        }
+        query = """
+                epv = [];
+                packages.each {
```

Comment:
This method is exactly same as base class `get_package_details_from_graph` except the `self.all_deps_without_pseudo`. I would recommend to refactor the get_package_details_from_graph and move out of class, parameters the package and ecosystem so that it could be reused in both base and derived.

Reply:
Ack! with different approach.